app/views/components/cases_cards.scala.html

```diff
@@ -93,20 +100,121 @@ <h3 class="card-heading govuk-heading-s">
             </div>
         </a>
     </div>
+    <h2 class="heading-medium">@messages("cases.menu.my-cases")</h2>
+    <div class="flex-container govuk-grid-row">
+        <a id="my-cases-id" class="link-text" href="my-cases">
+            <div class="card">
+                <h3 class="card-heading govuk-heading-s">@messages("operator.dashboard.classification.my-cases")</h3>
+                <p class="card-body">
+                    @if(countCases.getOrElse("my-cases", "0") == 1) {
+                        @messages("operator.dashboard.classification.my-cases.progress.singular", countCases.getOrElse("my-cases", "0"))
```

Comment:
can we use a single message key instead of passing in the number of the cases here if its displayed only when one case exists

Reply:
Good spot. We have now refactored it to use a single message key.